fs/serve_test.go

```diff
@@ -343,6 +343,10 @@ func (r *readFlags) Read(ctx context.Context, req *fuse.ReadRequest, resp *fuse.
 }
 
 func TestReadFileFlags(t *testing.T) {
+	if fuse.IsRunningUnderTravis {
+	  t.SkipNow()
```

Comment:
It would be good to document what the issue is, so others could continue to expand the tests.  If I comment out the skip and it works, I won't know why we see different behavior unless I also know what you saw.